src/test/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactoryTest.java

```diff
@@ -110,69 +110,69 @@ public void testCreateVariantFromVcfDeletionEmptyAlt() {
+        line = "1\t1000\trs123\tAT\tA\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1001, 1001, "T", ""));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\tGATC\tG\t.\t.\t.";
+        line = "1\t1000\trs123\tGATC\tG\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1001, 1003, "ATC", ""));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\t.\tATC\t.\t.\t.";
+        line = "1\t1000\trs123\t.\tATC\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1000, 1002, "", "ATC"));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\tA\tATC\t.\t.\t.";
+        line = "1\t1000\trs123\tA\tATC\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1001, 1002, "", "TC"));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\tAC\tACT\t.\t.\t.";
+        line = "1\t1000\trs123\tAC\tACT\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1002, 1002, "", "T"));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
         // Printing those that are not currently managed
-        line = "1\t1000\trs123\tAT\tT\t.\t.\t.";
+        line = "1\t1000\trs123\tAT\tT\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1000, 1000, "A", ""));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\tATC\tTC\t.\t.\t.";
+        line = "1\t1000\trs123\tATC\tTC\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1000, 1000, "A", ""));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\tATC\tAC\t.\t.\t.";
+        line = "1\t1000\trs123\tATC\tAC\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1001, 1001, "T", ""));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\tAC\tATC\t.\t.\t.";
+        line = "1\t1000\trs123\tAC\tATC\t.\t.\t.\tGT\t0/1";
         expResult = new LinkedList<>();
         expResult.add(new Variant("1", 1001, 1001, "", "T"));
         result = factory.create(FILE_ID, STUDY_ID, line);
         assertEquals(expResult, result);
 
-        line = "1\t1000\trs123\tATC\tGC\t.\t.\t.";
+        line = "1\t1000\trs123\tATC\tGC\t.\t.\t.\tGT\t0/1";
```

Comment:
That code looks fine. 
I went through the code to understand how are we generating the genotype information for each variant, in case of multi-allelic variants. 
According to data given, it should result in one variant being non variant, which is exactly what is happening. But, as there was no check for the non-variants, the test case was passing and started failing once we added the checks. 
I have corrected the test cases by changing the data and it started passing. This change is already part of the current PR